docker_env/README.md

```diff
@@ -0,0 +1,33 @@
+### TTSnake Docker execution environment 
+### Author: @Henrique1792
+### Date: 2021-11-15
+
+
+
+1. Rationale
+	After many installation workarounds pointed by colleagues, this suggestion
+	was created in order to easy up everyone on system dependencies.
+	It intends to allow execution without bloating host environment and solve
+	compatibility issues during execution.
+
+2. Usage
+	1. Get into `docker_env` directory and build it executing:
+		`docker build -t snake_env .`
+	2. After it's build, run container as follows (newbie's gentle execution):
+	`docker run -it --rm -v <snaskii21_path>:/data -w /data snake_env /bin/bash`
+	
+		Rationale:
+			`-it` stands for `iterative` mode (-i keeps STDIN attached and open, while -t allocate a pseudo-tty)
+			`--rm` cleans container up after execution (for small tests this flag is way useful to ignore)
+			`-v HOST_DIR:CONTAINER_DIR` sets a host directory share point inside the container
+			`-w WORKDIR_NAME` is your initial working directory after bring it up
```

Comment:
Found typo on line 105.
Also I thought line 112 was quite unclear and removed the last part of it (that can be just my opinion so I'm open to discussion 😛).

```suggestion
	2. After it's built, run container as follows (newbie's gentle execution):
	`docker run -it --rm -v <snaskii21_path>:/data -w /data snake_env /bin/bash`
	
		Rationale:
			`-it` stands for `iterative` mode (-i keeps STDIN attached and open, while -t allocate a pseudo-tty)
			`--rm` cleans container up after execution (for small tests this flag is way useful to ignore)
			`-v HOST_DIR:CONTAINER_DIR` sets a host directory share point inside the container
			`-w WORKDIR_NAME` is your initial working directory
```